opentelemetry/ext/honeycomb/__init__.py

```diff
@@ -81,6 +83,28 @@ def shutdown(self):
         self.client = None
 
 
+class HoneycombConsoleSpanExporter(SpanExporter):
+    """Honeycomb console span exporter for the Honeycomb AWS Lambda Instrumentation.
+    """
+
+    def __init__(
+        self,
+        service_name=None,
+        out=sys.stdout,
+        formatter=lambda d: json.dumps(d) + os.linesep,
+    ):
+        self.out = out
+        self.formatter = formatter
+        self.service_name = service_name
+
+    def export(self, spans):
+        for d in _translate_to_hny(spans):
+            del d["start_time"]  # trust API log timestamp?
```

Comment:
If we fix honeycombio/honeycomb-lambda-extension#11 by trusting a timestamp field and ignoring any duration field, we'll need to adjust this line.